[2022-12-04T16:13:26.531+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-03T19:00:00+00:00 [queued]>
[2022-12-04T16:13:26.539+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-03T19:00:00+00:00 [queued]>
[2022-12-04T16:13:26.540+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:13:26.541+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:13:26.542+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:13:26.552+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): generate_data> on 2022-12-03 19:00:00+00:00
[2022-12-04T16:13:26.561+0000] {standard_task_runner.py:55} INFO - Started process 393 to run task
[2022-12-04T16:13:26.568+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'generate_data', 'scheduled__2022-12-03T19:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmpyj2awxm_']
[2022-12-04T16:13:26.573+0000] {standard_task_runner.py:83} INFO - Job 41: Subtask generate_data
[2022-12-04T16:13:26.677+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.generate_data scheduled__2022-12-03T19:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:13:26.760+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=generate_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-03T19:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-03T19:00:00+00:00
[2022-12-04T16:13:27.128+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-04T16:13:27.142+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=generate_data, execution_date=20221203T190000, start_date=20221204T161326, end_date=20221204T161327
[2022-12-04T16:13:27.184+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:13:27.212+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
